data/src/main/java/com/android254/data/network/apis/SponsorsApi.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android254.data.network.apis
+
+import com.android254.data.network.Constants
+import com.android254.data.network.models.responses.SponsorsPagedResponse
+import com.android254.data.network.util.dataResultSafeApiCall
+import com.android254.domain.models.DataResult
+import io.ktor.client.HttpClient
+import io.ktor.client.call.body
+import io.ktor.client.request.get
+import javax.inject.Inject
+
+
+class SponsorsApi @Inject constructor(
+    private val client: HttpClient
+) {
+
+    suspend fun fetchSponsors(perPage: Int, page: Int): DataResult<SponsorsPagedResponse> =
+       dataResultSafeApiCall{
+            val response = client.get("${Constants.BASE_URL}/events/droidconke-2022-797/feeds?per_page=$perPage&page=$page")
```

Comment:
Can you inline this?

```suggestion
           client.get("${Constants.BASE_URL}/events/droidconke-2022-797/feeds?per_page=$perPage&page=$page").body()
```

Reply:
This has been updated

Reply:
@jumaallan Can you check if this is now resolved?